packages/multi-test/src/app.rs

```diff
@@ -835,14 +1035,17 @@ mod test {
         // TODO: check error?
     }
 
-    fn query_router<ExecC, QueryC>(
-        router: &Router<ExecC, QueryC>,
+    fn query_router<BankT, CustomT, WasmT>(
+        router: &Router<BankT, CustomT, WasmT>,
         api: &dyn Api,
```

Comment:
Huh, there is still &dyn Api, but you fix the type in App?

Okay, so Router doesn't care about it as an argument, but App does when storing it?